### DISCUSSION TITLE
heroku alternatives

### DISCUSSION THREAD

**Discussion by ILikeBypassing** [3kh0/ext-remover#238] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
so ive been using a personal proxy deployed to heroku, but as you may know, heroku isnt free anymore as of nov 28.
if anyone knows where i can deploy a github repo just like heroku please tell me.

**Comment by frogmasamune** [comment, marked as answer]
replit, netlify, glitch.me, vercel.app, and github.io domains

**Reply by ILikeBypassing** [reply]
aight thanks

**Reply by Terrinn** [reply]
do you have any proxies on those websites?

**Reply by ILikeBypassing** [reply]
just delpoy your own using the holy unblocker git repo

**Comment by FilianFromPhotop** [comment]
How do you launch a private proxy?

**Reply by ILikeBypassing** [reply]
i use holy unblocker. so basically you download the source code from git and import it to any git repo deployer (thats what im looking for) and you just view your app that you plagiarized

**Comment by CharanKuberns** [comment]
After heroku ended its free trial, developers are looking for cost effective and simple solution to deploy their projects and I would suggest you try kuberns.com. It's an agentic AI platform that helps you deploy in one click. Read more to get more information about best heroku [alternatives.](**https://kuberns.com/blogs/post/the-ultimate-guide-to-heroku-alternatives-in-2025/**)
